### PR TITLE
Markdown Previewer using Vue JS

### PR DESCRIPTION
**Pull request by Bishal-35**
# Pull Request for WebMasterLog 💡

## Issue Title: Markdown Previewer using Vue JS

- **Name of the project** : Markdown Previewer
- **Category:** VueJS, HTML and CSS
- **Name:** Bishal Karmakar
- **GitHub ID:** https://github.com/Bishal-35
- **Idenitfy yourself:** Developer

Closes: #59 

### Describe the add-ons or changes you've made 📃

I have created a web application which converts markdown code to readable code in real time using VueJS, HTML and CSS

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷
Screenshot 2024-02-18 210010

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Directory name should be `Markdown-Previewer` with hyphen. Change it's name. Also follow the updated README TEMPLATE of the repository.

**Comment by Avdhesh-Varshney** [comment]
@Bishal-35 Also fill the pr template and link your issue with it.

**Comment by Bishal-35** [comment]
@Avdhesh-Varshney I have changed all the requested changes

File: Vue-JS-Projects/Basic/Markdown-Previewer/Readme.md
Comment (Avdhesh-Varshney) [inline Vue-JS-Projects/Basic/Markdown-Previewer/Readme.md:1]: @Bishal-35  Follow this [README](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/ccfcdca6d08efd69961187762ea2707d02def086/.github/PROJECT-README-TEMPLATE.md) template.

**Comment by Bishal-35** [comment]
shall i add working video?

**Comment by Bishal-35** [comment]
@Avdhesh-Varshney check if it is okay now

File: Vue-JS-Projects/Basic/Markdown-Previewer/Readme.md
Comment (Avdhesh-Varshney) [reply Vue-JS-Projects/Basic/Markdown-Previewer/Readme.md:1]: @Bishal-35 go through this readme file. Click on this [Link](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/ccfcdca6d08efd69961187762ea2707d02def086/.github/PROJECT-README-TEMPLATE.md)

https://github.com/Avdhesh-Varshney/WebMasterLog/blob/ccfcdca6d08efd69961187762ea2707d02def086/.github/PROJECT-README-TEMPLATE.md

**Comment by Bishal-35** [comment]
Shall I have to create a working video?

**Comment by Avdhesh-Varshney** [comment]
@Bishal-35 If possible add the video, otherwise remove the heading if you will not add. OK

**Comment by Bishal-35** [comment]
Okay

**Comment by Bishal-35** [comment]
@Avdhesh-Varshney check it now

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Comment by Bishal-35** [comment]
Thankyou.
Please assign me another issue

**Pull request merged**